Author: Michael R. Crusoe <crusoe@debian.org>
Description: Remove tokio feature
Forwarded: not-needed

tokio-io is not yet completely packaged for Debian
--- liblzma.orig/Cargo.toml
+++ liblzma/Cargo.toml
@@ -43,7 +43,6 @@
 
 [package.metadata.docs.rs]
 features = [
-    "tokio-io",
     "futures",
     "parallel",
 ]
@@ -57,10 +56,6 @@
 path = "tests/drop-incomplete.rs"
 
 [[test]]
-name = "tokio"
-path = "tests/tokio.rs"
-
-[[test]]
 name = "xz"
 path = "tests/xz.rs"
 
@@ -76,10 +71,6 @@
 version = "1.16.0"
 optional = true
 
-[dependencies.tokio-io]
-version = "0.1.12"
-optional = true
-
 [dev-dependencies.quickcheck]
 version = "1.0.1"
 
@@ -94,10 +85,3 @@
     "num_cpus",
 ]
 static = ["liblzma-sys/static"]
-tokio = [
-    "tokio-io",
-    "futures",
-]
-
-[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies.tokio-core]
-version = "0.1.17"
